content/doc/book/installing/servlet-containers.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: section
+title: Other Servlet Containers
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+Jenkins is typically run as a standalone application in its own process.
+The Jenkins WAR file bundles link:https://github.com/jenkinsci/winstone[Winstone],
+a link:https://www.eclipse.org/jetty/[Jetty] servlet container wrapper,
+and can be started on any operating system or platform with a version of Java supported by Jenkins.
+This is the preferred way to deploy Jenkins and is fully supported.
+
+Theoretically, Jenkins can also be run as a servlet in a traditional servlet container
+like link:https://tomcat.apache.org/[Apache Tomcat] or link:https://www.wildfly.org/[WildFly],
+but in practice this is largely untested and there are many caveats.
+In particular, support for WebSocket agents is only implemented for the Jetty servlet container.
+See the link:/doc/administration/requirements/servlet-containers[Servlet Container Support Policy] page for details.
+
+WARNING: Support for traditional servlet containers may be discontinued in the future.
+
+Jenkins requires Servlet API 4.0 (Jakarta EE 8) with `javax.servlet` imports.
+Jenkins is incompatible with Servlet API 5.0 (Jakarta EE 9) or later with `jakarta.servlet` imports.
+Ensure that the Servlet API version of your chosen servlet container is compatible before running Jenkins.
```

Comment:
Since this text is identical to the other document in the admin section: can we use AsciiDoc includes in our installation to avoid that duplication?

Reply:
Not sure offhand